src/content/add-to-app/ios/project-setup.md

```diff
@@ -273,89 +272,95 @@ some/path/MyApp/
+To use the simulator, embed the Debug version of the Flutter frameworks
+in your Debug build configuration.
+To do this, use `$(PROJECT_DIR)/Flutter/$(CONFIGURATION)`
+in the **Framework Search Paths** (`FRAMEWORK_SEARCH_PATHS`) build setting.
+This embeds the Release frameworks in the Release configuration,
 and the Debug frameworks in the Debug Configuration.
 
-You must also open `MyApp.xcodeproj/project.pbxproj` (from Finder) 
+You must also open `MyApp.xcodeproj/project.pbxproj` (from Finder)
 and replace `path = Flutter/Release/example.xcframework;`
 with `path = "Flutter/$(CONFIGURATION)/example.xcframework";`
-for all added frameworks. (Note the added `"`.)
+for all added frameworks. The double quotes (`"`) are necessary.
 :::
 
-#### Embed the frameworks
+##### Embed the frameworks
 
-The generated dynamic frameworks must be embedded
-into your app to be loaded at runtime.
+To load the generated dynamic frameworks at runtime,
+embed them into your app.
 
 :::important
 Plugins might produce [static or dynamic frameworks][].
 Static frameworks should be linked on, but never embedded.
-If you embed a static framework into your application,
-your application is not publishable to the App Store
-and fails with a
+If you embed a static framework into your iOS app,
+you can't publish that app to the App Store.
+Publishing fails with a
 **Found an unexpected Mach-O header code** archive error.
 :::
 
-After linking the frameworks, you should see them in the 
+After linking the frameworks, they should display in the
 **Frameworks, Libraries, and Embedded Content**
-section of your target's **General** settings. 
-To embed the dynamic frameworks
-select **Embed & Sign**. 
+section of your target's **General** settings.
 
-They will then appear under **Embed Frameworks** within 
-**Build Phases** as follows: 
+To embed the dynamic frameworks, select **Embed & Sign**.
+They then appear under **Embed Frameworks** within **Build Phases** as shown.
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/embed-xcode.png", alt:"Embed frameworks in Xcode" %}
+{% render docs/captioned-image.liquid, image:"development/add-to-app/ios/project-setup/embed-xcode.png", caption:"Embed frameworks in Xcode" %}
 
 You should now be able to build the project in Xcode using `⌘B`.
 
-### Option C - Embed application and plugin frameworks in Xcode and Flutter framework with CocoaPods
+### Method C - Embed app and plugin frameworks in Xcode; Flutter framework with CocoaPods {:#method-c}
+
+#### Requirements {:#method-c-reqs}
 
-Alternatively, instead of distributing the large Flutter.xcframework
-to other developers, machines, or continuous integration systems,
-you can instead generate Flutter as CocoaPods podspec by adding
-the flag `--cocoapods`. This produces a `Flutter.podspec`
-instead of an engine Flutter.xcframework.
-The App.xcframework and plugin frameworks are generated
+This method has the same requirements as Method B.
```

Comment:
Could we inline the requirements to this section? The sidebar doesn't have a link to Method B's requirements.